mosaic_api/README.md

```diff
@@ -0,0 +1,104 @@
+# Mosaic API
+
+An API to proxy STAC API searches into a MosaicJSON endpoint.
```

Comment:
We might want to enumerate the endpoints we are creating (e.g. POST and GET mosaicjson) to make it clear we're not run a dynamic tiler for mosaicjson (yet!)